cmd/chronoctl/migrate.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/kv"
+	"github.com/influxdata/chronograf/kv/bolt"
+	"github.com/influxdata/chronograf/kv/etcd"
+)
+
+func init() {
+	parser.AddCommand("migrate",
+		"Migrate db (beta)",
+		"The migrate command (beta) will copy one db to another",
+		&migrateCommand{})
+}
+
+type migrateCommand struct {
+	From string `short:"f" long:"from" description:"Full path to boltDB file or etcd (e.g. 'bolt:///path/to/chronograf-v1.db' or 'etcd://user:pass@localhost:2379" default:"chronograf-v1.db"`
+	To   string `short:"t" long:"to" description:"Full path to boltDB file or etcd (e.g. 'bolt:///path/to/chronograf-v1.db' or 'etcd://user:pass@localhost:2379" default:"etcd://localhost:2379"`
```

Comment:
I'm quite on the fence about having these defaults. It seems dangerous to write data without more "intention" from the user, so I may remove the default for `to`, requiring the user to specify where they want to migrate their data to.